lumicks/pylake/tests/test_channels/test_arithmetic.py

```diff
@@ -52,7 +53,7 @@ def test_operator(first_slice, second_slice, operation):
         # With slices, force calibration should never be carried over (since it's invalid)
         assert not getattr(first_slice, operation)(second_slice).calibration
 
-    for operator, _ in operators:
+    for operator, _, _, _ in operators:
```

Comment:
could also just do `*_`  at the end or did you want to keep the specific placeholders for each argument in the unpack?

Reply:
Good point. Applying that everywhere in that file.